SXR/Extensions/MixedReality/src/main/java/com/samsungxr/mixedreality/SXRMixedReality.java

```diff
@@ -136,6 +136,11 @@ public SXRMixedReality(SXRScene scene, int platform, boolean enableCloudAnchor)
      */
     public void selectARPlatform(int platform, boolean enableCloudAnchor)
     {
+        String prop = System.getProperty("debug.samsungxr.hmt");
```

Comment:
You need to use our SystemPropertyUtil.getSystemPropertyString helper instead to read the android property. System.getProperty reads a Java "system" property which is something else.